### ISSUE TITLE
Can't log on

### ISSUE THREAD

**Issue by boazdori** [OliBlade/3Cols#28]
I am trying to log on and getting the following error message

This page isn’t working right now
If the problem continues, contact the site owner.
HTTP ERROR 400

**Comment by OliBlade** [comment]
Hi boazdori,

3Cols seems to be allowing us to log in fine. What browser are you using? 

Here's a few things to try: 

- Clearing cache for the site
- Incognito window 
- Make sure the time/ time zone on your device is set correctly as this can cause issues with Google Authentication

**Comment by David-Carty** [comment]
For the first time, in all the time I have been using 3Cols I have found that I cannot log in and I am receiving HTTP Error 400.  Yes, I have cleared cache etc.  In UK, time is correct and I am using Google Chrome Version 97.0.4692.71 on Windows 11.

**Comment by David-Carty** [comment]
I was able to log in on another laptop.  This laptop had Windows 10 on it and Chrome: Version 96.0.4664.45  (64-bit)

**Comment by OliBlade** [comment]
Yep looks like you're right, something about using Chromium 97 is causing us issues

**Comment by OliBlade** [comment]
We just rolled out a major update which included a fix for this issue. You might need to give the site a refresh after you log in. Please let us know how this goes

**Comment by David-Carty** [comment]
Looks good, I'm back in on the latest version of Chrome. Cheers @OliBlade
